portal/portal-sprint-102.md

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
The images in the article I think are too large, overshadowing every text and paragraph.
Luckily, we can adjust the image size in markdown:
https://stackoverflow.com/questions/24383700/resize-image-in-the-wiki-of-github-using-markdown

Reply:
👍